ci/build.ps1

```diff
@@ -12,6 +13,7 @@ $nvimCmakeVars = @{
   CMAKE_BUILD_TYPE = $cmakeBuildType;
   BUSTED_OUTPUT_TYPE = 'nvim';
 }
+$uploadToCodeCov = $false
```

Comment:
we check `if ($uploadToCodeCov)` which should resolve to false if the variable is undefined ... although that might not work in strict mode. https://stackoverflow.com/a/9181540/152142